packages/cf.js/src/types/node-protocol.ts

```diff
@@ -0,0 +1,105 @@
+import { BigNumber } from "ethers/utils";
```

Comment:
Yeah, i was looking at namespacing. Should probably use them, prefixing "Node" for each type is redundant.

Reply:
Source?

Reply:
I deleted the comment, because I misread the context. If we restrict usage of namespaces to types and not actual implementation code, I don't see a problem.

Reply:
:+1: